Makefile.am

```diff
@@ -250,10 +250,12 @@ install-data-local:: $(WEBPACK_INSTALL)
 install-data-local:: $(wildcard $(top_srcdir)/dist/*/*.map)
 	$(V_TAR) tar -c --transform="s@.*dist/@$(debugdir)$(pkgdatadir)/@" -T/dev/null $^ | tar --no-same-owner -C $(DESTDIR)/ -xv
 uninstall-local::
-	find $(DESTDIR)$(pkgdatadir) -ignore_readdir_race -type f -exec rm -f {} \;
-	find $(DESTDIR)$(pkgdatadir) -ignore_readdir_race -type d -empty -delete
-	find $(DESTDIR)$(debugdir)$(pkgdatadir) -ignore_readdir_race -type f -delete
-	find $(DESTDIR)$(debugdir)$(pkgdatadir) -ignore_readdir_race -type d -empty -delete
+	test ! -d $(DESTDIR)$(pkgdatadir) || \
```

Comment:
Feels a bit strange that we need this bit, as this was only about debug files.

Reply:
> Add a check also to the equivalent rule for the normal installation
files: this makes make uninstall nicely idempotent.